.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+node_js:
+  - "10"
+  - node
+sudo: false
+ 
+branches:
+  only:
+  - "staging"
+ 
+cache:
+  directories:
+    - node_modules
+ 
+deploy:
+  provider: heroku
+  skip_cleanup: true
+  keep-history: true
+  api_key: 2de18260-de86-45e7-a867-be5049c39897
```

Comment:
try to find a way to hide you api_key, to minimize exposure to attack.

Reply:
Thank you.
I revoked the API key. And am looking into the docs to find different ways.